src/views/Widget/TabContent/Panes/Operate/OperatePane.tsx

```diff
@@ -85,19 +86,23 @@ const OperatePane: React.FC<OperatePaneProps> = (props) => {
   }, [formValues, handleInputChange]);
 
   //todo
-  const formatAmount2Withdraw = (ev: React.ChangeEvent<HTMLInputElement>) => {
+  const formatAmount = (ev: React.ChangeEvent<HTMLInputElement>) => {
     let amount = ev.target.value;
-
-    setAmount2Withdraw(amount);
+    setAmount(amount);
   };
 
   //todo
   const unstake = async () => {
-    await unstakeAndClaim(signer, monitorTx, selectedCrucible, amount2Withdraw);
+    await unstakeAndClaim(signer, monitorTx, selectedCrucible, amount);
     setModalIsOpen(false);
   };
+  const increaseStake = async () => {
```

Comment:
@corollari Increase stake functionality is here

Reply:
Hmm I was referring to the code that interacts with the contracts, AFAIK mintAndLock will mint a crucible instead of funding a new one (which is what we are trying to do, no?)